src/app/stepper/stepper.component.scss

```
@@ -0,0 +1,94 @@
+.stepper-container {
+  width: 100%;
+  margin-top: 40px;
+  margin-bottom: 40px;
+
+  .stepper-line {
+    display: flex;
+    justify-content: space-between;
+
+    mat-label {
+      box-sizing: border-box;
+      display: flex;
+      font-family: Inter, sans-serif;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 14px;
+      line-height: 17px;
+      text-align: center;
+      padding-top: 13px;
+      padding-left: 18px;
```

Numbers inside the stepper look a little bit off-center when aligned this way, so since you are using flex here, you can center them like this
```suggestion
      align-items: center;
      justify-content: center;
```